source/blog/2022-11-17-announcing-hanami-200rc.html.markdown

```diff
@@ -0,0 +1,362 @@
+
+```ruby
+Admin::Slice["feeds.process_feed"] # => #<Feeds::ProcessFeed>
+
+# or as a dep:
+module Admin
+  class SomeClass
+    include Deps["feeds.process_feed"]
+  end
+end
+```
+
+Slices can also be mounted within the router:
+
+```ruby
+module Bookshelf
+  class Routes < Hanami::Routes
+    slice :admin, at: "/admin" do
+      get "/books" to: "books.index"
+    end
+  end
+end
+```
+
+Slices can also be selectively loaded at boot time, which brings great advantages for ensuring best performance of certain production workloads. If you had a process working a Sidekiq queue for jobs from the feeds slice only, then you can set `HANAMI_SLICES=feeds` to load that slice only, giving you code isolation and optimal memory and boot time performance for that process.
+
+Working with slices helps you maintain a clear understanding of the relationship between your app’s high-level concerns, just like how the Deps helps with this for individual components. Slices are the key to helping your app be just as easy to maintain at day 1,000 as it was at day 1.
+
+## Getting started with Hanami 2.0
+
+Hopefully by now you’re excited to give Hanami 2.0 a try! To help you on your way, we’ve completely rewritten our friendly [user guides][guides] to cover all the important concepts for this new release.
+
+A huge thank you to [Andrew Croome][andrew] and [Seb Wilgosz][seb] for delivering these guides!
+
+[guides]: https://guides.hanamirb.org/v2.0/introduction/getting-started/
+[andrew]: http://github.com/andrewcroome
+[seb]: https://github.com/swilgosz
+
+## What’s included?
+
+Today’s release includes the following gems:
+
+- hanami v2.0.0
+- hanami-cli v2.0.0
+- hanami-controller v2.0.0
+- hanami-router v2.0.0
+- hanami-validations v2.0.0
+- hanami-utils v2.0.0
+- hanami-reloader v2.0.0
+- hanami-rspec v2.0.0
+
+For specific changes, please see each gem’s own CHANGELOG and FEATURES files.
+
+## How can I try it?
+
+```shell
+$ gem install hanami --pre
+$ hanami new bookshelf
+$ cd bookshelf
+$ bundle exec hanami --help
+```
+
+## What’s next
+
+For this release, we’ve focused on developing a powerful, polished framework core along with all the tools you need to deliver great web APIs. We hope you enjoy it!
+
+From here we’re going to focus on finishing the rest of the “full stack” Hanami experience. This will include database integration, views, helpers and front end assets. We’re committed to delivering these within the first quarter of 2023.
+
+## Contributors
+
+Thank you to these wonderful people for contributing to Hanami 2.0!
+
+- [Andrew Croome](https://github.com/andrewcroome)
+- [Benjamin Klotz](https://github.com/tak1n)
+- [Luca Guidi](https://github.com/jodosha)
+- [Lucas Mendelowski](https://github.com/lcmen)
+- [Marc Busqué](https://github.com/waiting-for-dev)
+- [Narinda Reeders](https://github.com/narinda)
+- [Pat Allen](https://github.com/pat)
+- [Peter Solnica](https://github.com/solnic)
+- [Philip Arndt](https://github.com/parndt/)
```

Comment:
```suggestion
- [Philip Arndt](https://github.com/parndt)
```

the others don't have a trailing /

also ❤️❤️❤️

Reply:
You're special, Phil

Reply:
I knew it!